SimPEG/utils/mat_utils.py

```diff
@@ -409,12 +409,8 @@ def coterminal(theta):
         Coterminal angles
 
     """
-    sub = theta[np.abs(theta) >= np.pi]
-    sub = -np.sign(sub) * (2 * np.pi - np.abs(sub))
-
-    theta[np.abs(theta) >= np.pi] = sub
-
-    return theta
+    coterminal = np.sign(theta) * (np.abs(theta) % np.pi)
+    return coterminal
```

Comment:
I think we should move forward with this implementation:
```suggestion
    return (theta + np.pi) % (2 * np.pi) - np.pi
```